projects/ng-wizard/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```

Comment:
No ivy? That is like the big thing in Angular 9...

Reply:
It is not recommended to use Ivy when publishing as npm. [angular.io](https://angular.io/guide/creating-libraries#publishing-your-library)